packages/fxa-auth-server/test/chaiWithoutTruncation.js

```diff
@@ -0,0 +1,5 @@
+const chai = require('chai');
```

Comment:
You could add a license to the top of this file, and we've got a sorta rule of thumb that for new files, let's go ahead and create them as `.ts` instead, if possible.

Reply:
Ah great idea will do!